Description: Do not install obsolete headers.
 These header files are no longer compilable after source refactoring.
Author: A. Maitland Bottoms <bottoms@debian.org>

--- a/gr-atsc/include/gnuradio/atsc/CMakeLists.txt
+++ b/gr-atsc/include/gnuradio/atsc/CMakeLists.txt
@@ -41,7 +41,6 @@
     equalizer_lms2_impl.h
     equalizer_lms_impl.h
     equalizer_nop_impl.h
-    exp2_lp_impl.h
     fake_single_viterbi_impl.h
     field_sync_demux.h
     field_sync_mux.h
@@ -51,29 +50,6 @@
     fs_checker_naive_impl.h
     fs_correlator_impl.h
     fs_correlator_naive_impl.h
-    GrAtscBitTimingLoop2.h
-    GrAtscBitTimingLoop3.h
-    GrAtscBitTimingLoop.h
-    GrAtscConvert2xTo20.h
-    GrAtscDataSegToSoftDataSeg.h
-    GrAtscDeinterleaver.h
-    GrAtscDerandomizer.h
-    GrAtscEqualizer.h
-    GrAtscFieldSyncChecker.h
-    GrAtscFieldSyncCorrelator.h
-    GrAtscFieldSyncDemux.h
-    GrAtscFieldSyncMux.h
-    GrAtscFPLL.h
-    GrAtscInterleaver.h
-    GrAtscRandomizer.h
-    GrAtscRSDecoder.h
-    GrAtscRSEncoder.h
-    GrAtscSegSymSync.h
-    GrAtscSegSymSyncImpl_export.h
-    GrAtscSegSymSyncImpl.h
-    GrAtscSymbolMapper.h
-    GrAtscTrellisEncoder.h
-    GrAtscViterbiDecoder.h
     interleaver_fifo.h
     interleaver.h
     pad.h
@@ -81,12 +57,9 @@
     randomizer.h
     randomizer_impl.h
     reed_solomon_impl.h
-    root_raised_cosine_bandpass_impl.h
-    root_raised_cosine_impl.h
     rs_decoder.h
     rs_encoder.h
     single_viterbi_impl.h
-    slicer_agc_impl.h
     sliding_correlator_impl.h
     sssr_impl.h
     syminfo_impl.h
@@ -96,7 +69,6 @@
     types.h
     viterbi_decoder.h
     viterbi_decoder_impl.h
-    vsbtx_lp_impl.h
     DESTINATION ${GR_INCLUDE_DIR}/gnuradio/atsc
     COMPONENT "atsc_devel"
 )
--- a/gr-qtgui/include/gnuradio/qtgui/CMakeLists.txt
+++ b/gr-qtgui/include/gnuradio/qtgui/CMakeLists.txt
@@ -45,8 +45,6 @@
   qtgui_types.h
   sink_c.h
   sink_f.h
-  spectrumdisplayform.h
-  SpectrumGUIClass.h
   spectrumUpdateEvents.h
   timedisplayform.h
   timecontrolpanel.h
